OpenRA.Mods.Common/Widgets/SupportPowerTimerWidget.cs

```diff
@@ -46,20 +79,77 @@ public override void Tick()
 			}).ToArray();
 		}
 
-		public override void Draw()
+		public int DrawAllies(int y)
```

Comment:
There's a bit too much repetition across this method and DrawEnemies for my liking.